igor_src/Makefile.am

```
@@ -1,17 +1,38 @@
 ACLOCAL_AMFLAGS = -I ../m4
 
 # Declare program name and install it in $(bindir) directory 
-bin_PROGRAMS = igor
+bin_PROGRAMS = igor #testIgor TestAlign TestCDR3 TestCDR3AnchorReads
```

Just cleaning a bit your testing stuff
```suggestion
bin_PROGRAMS = igor 
```